backend/serializers.py

```diff
@@ -86,8 +76,8 @@ class Meta(BaseOrderSerializer.Meta):
 class RestrictiveUpdateOrderWithOrderItemsSerializer(BaseOrderWithOrderItemsSerializer):  # noqa
     class Meta(BaseOrderWithOrderItemsSerializer.Meta):
         read_only_fields = \
-            BaseOrderWithOrderItemsSerializer.Meta.read_only_fields + \
-            ('event',)
+            BaseOrderWithOrderItemsSerializer.Meta.read_only_fields
+# + \('event',)
```

Comment:
Shouldn't it be possible to remove this completely?

Reply:
That one's on me, forgot adding a todo comment to that line when testing it.